cg/services/orders/submitters/order_submitter.py

```diff
@@ -0,0 +1,43 @@
+"""Abstract base classes for order submitters."""
+
+from abc import ABC, abstractmethod
+
+from cg.models.orders.order import OrderIn
+from cg.services.orders.order_lims_service.order_lims_service import OrderLimsService
+from cg.store.store import Store
+
+
+class ValidateOrderService(ABC):
+    @abstractmethod
+    def __init__(self, status_db: Store):
+        self.status_db = status_db
+
+    @abstractmethod
+    def validate_order(self, order_in: OrderIn):
+        pass
+
+
+class StoreOrderService(ABC):
```

Comment:
Should this not be in the store_order_services folder?

Reply:
Yes, it was here for now just to display the overall design